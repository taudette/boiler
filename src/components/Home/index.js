import React from 'react';

export const Home = (props) => (
  <div>{props.testData}</div>
)

export default Home
