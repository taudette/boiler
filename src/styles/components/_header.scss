@import '../base/_settings.scss';

.header {
  color: $blue;
}