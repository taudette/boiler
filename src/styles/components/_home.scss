@import '../base/_settings.scss';

* {
  color: $blue;
}