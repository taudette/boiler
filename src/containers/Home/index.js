import React from 'react';
import HomeComponent from '../../components/Home'
import '../../styles/components/_home.scss'

export const Home = () => (
  <div>
    <HomeComponent />
  </div>
)

export default Home
