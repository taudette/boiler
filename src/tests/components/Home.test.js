import React from 'react';
import { shallow } from 'enzyme';
import { Home } from '../../components/Home';
import testData from '../fixtures/home';

test('should render test data from props', () => {
  const wrapper = shallow(<Home testData={testData} />);
  expect(wrapper).toMatchSnapshot();
})

test('should render with no props', () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
})