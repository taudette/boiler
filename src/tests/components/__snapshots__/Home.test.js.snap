// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render test data from props 1`] = `
<div>
  test-data
</div>
`;

exports[`should render with no props 1`] = `<div />`;
